scripts/rv_expression_association/get_rv_outliers.py

```diff
@@ -0,0 +1,166 @@
+import click
+import logging
+import hail as hl
+import pandas as pd
+from cpg_utils.hail_batch import output_path
+from cloudpathlib import AnyPath
+
+# use logging to print statements, display at info level
+logging.basicConfig(format='%(levelname)s:%(message)s', level=logging.INFO)
+
+
+# get OneK1K sample ID and gene name as arguments using click
+@click.command()
+@click.option('--onek1k-id', required=True)
+@click.option('--gene-name', required=True)
+@click.option('--chrom', required=True)
+def main(
+    onek1k_id: str,
+    gene_name: str,
+    chrom: str,
+):
+    """for a given individual,
+    and gene for which that individual is an expression outlier
+    get all variants within a window,
+
+    then filter for variants that:
+    - are alt (0/1 or 1/1) for that individual
+    - are biallelic SNVs
+    - have regulatory consequences based on VEP annotations
+
+    also annotate variants with the following:
+    - CADD score
+    - MAF within the OneK1K sample
+    - MAF in gnomad
+
+    create the following table:
+    donor ID | gene ID | variant ID | position | CADD | MAF (OneK1K) | MAF (gnomad) | ..
+    """
+
+    # file matching OneK1K IDs to CPG (internal) and TOB (external) IDs
+    sample_key_filename = (
+        'gs://cpg-tob-wgs-test/scrna-seq/grch38_association_files/OneK1K_CPG_IDs.tsv'
+    )
+    sample_key_df = pd.read_csv(sample_key_filename, sep='\t', index_col=0)
```

Comment:
Can you test-run this to make sure the syntax works? I don't think pandas will be able to automatically resolve gs:// paths, but I could be wrong..